src/azure-cli-core/azure/cli/core/parser.py

```diff
@@ -392,135 +391,165 @@ def _get_extension_run_after_dynamic_install_config(self):
                                                                   default_value) if cli_ctx else default_value
         return run_after_extension_installed
 
-    def _check_value(self, action, value):  # pylint: disable=too-many-statements, too-many-locals, too-many-branches
+    def _check_value_in_extensions(self, cli_ctx, args, no_prompt):  # pylint: disable=too-many-statements, too-many-locals
```

Comment:
Can we move this function to a separate file?

Reply:
Done